tests/operators/test_local.py

```diff
@@ -603,6 +686,8 @@ def test_run_operator_dataset_url_encoded_names(caplog):
     ]
 
 
+# TODO: Make test compatible with Airflow 3.0. Issue:https://github.com/astronomer/astronomer-cosmos/issues/1705
+@pytest.mark.skipif(version.parse(airflow_version).major == 3, reason="Test need to be updated for Airflow 3.0")
```

Comment:
Has this test started failing now?

Reply:
It was probably a rebase issue, I'll re-add it!

Reply:
Addressed in 2a113a5